.github/workflows/assign-default-project.yml

```diff
@@ -0,0 +1,23 @@
+name: Assign default project to issues
+
+on:
+  issues:
+    types:
+    - opened
```

Comment:
we could also support more use cases when a label is added to an existing issue, not sure it's needed at this stage. If we want to support it, `peter-evans/create-or-update-project-card` might not be the best choice for a GH action because there is no way to control the "update" behaviour (might fork it to make it controllable?)